src/components/AsideHeader/components/CompositeBar/Item/Item.module.scss

```
@@ -289,7 +303,10 @@ $actionLeftRightMargin: 10px;
                     var(--_--item-selected-background-color)
                 );
 
-                border-radius: var(--gn-aside-header-item-expanded-radius);
+                border-radius: var(
+                    --gn-aside-header-item-expanded-radius,
+                    var(--_--item-expanded-radius)
+                );
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)